app/frontend/src/components/forms/manage/DocumentTemplate.vue

```diff
@@ -0,0 +1,372 @@
+<script>
```

Comment:
I think @jasonchung1871 and/or @nimya-aot would be able to help with how to write tests for this. I'm not sure what the goal/approach is for testing the frontend.